src/form/Calendar/Calendar.story.tsx

```diff
@@ -119,11 +118,13 @@ export const Multiview = () => {
 
   return (
     <Card>
-      <CalendarRange
+      <Calendar
         value={range}
         onChange={val => setRange(val as [Date, Date])}
-        numMonths={3}
-        enableDayOfWeek
+        numMonths={2}
```

Comment:
@steppy452 - I imagined `CalendarRange` to be the top most component, not the calendar.

Reply:
so I tried that - it basically just duplicated the calendar code and added the same logic .. unless you don't think we need the ability to switch into month/year views